buildnotifylib/generated/server_configuration_ui.py

```diff
@@ -127,23 +146,30 @@ def setupUi(self, serverConfigurationDialog):
         self.addServerUrl.returnPressed.connect(self.loadUrlButton.click)
         self.submitButton.clicked.connect(serverConfigurationDialog.accept)
         QtCore.QMetaObject.connectSlotsByName(serverConfigurationDialog)
-        serverConfigurationDialog.setTabOrder(self.addServerUrl, self.loadUrlButton)
-        serverConfigurationDialog.setTabOrder(self.loadUrlButton, self.submitButton)
-        serverConfigurationDialog.setTabOrder(self.submitButton, self.projectsList)
+        serverConfigurationDialog.setTabOrder(self.addServerUrl, self.projectsList)
 
     def retranslateUi(self, serverConfigurationDialog):
         _translate = QtCore.QCoreApplication.translate
         serverConfigurationDialog.setWindowTitle(_translate("serverConfigurationDialog", "Add Server"))
         self.cctrayUrlLabel.setText(_translate("serverConfigurationDialog", "Path to cctray.xml"))
-        self.loadUrlButton.setText(_translate("serverConfigurationDialog", "Load"))
         self.authenticationSettings.setTitle(_translate("serverConfigurationDialog", "Authentication"))
         self.usernameLabel.setText(_translate("serverConfigurationDialog", "Username"))
         self.passwordLabel.setText(_translate("serverConfigurationDialog", "Password"))
         self.groupBox_2.setTitle(_translate("serverConfigurationDialog", "Misc"))
         self.timezoneLabel.setText(_translate("serverConfigurationDialog", "Server timezone"))
         self.displayPrefixLabel.setText(_translate("serverConfigurationDialog", "Display prefix"))
         self.displayPrefix.setPlaceholderText(_translate("serverConfigurationDialog", "e.g. branch/release"))
+        self.loadUrlButton.setText(_translate("serverConfigurationDialog", "Load"))
         self.chooseProjectsLabel.setText(_translate("serverConfigurationDialog", "Choose projects"))
         self.backButton.setText(_translate("serverConfigurationDialog", "Back"))
         self.submitButton.setText(_translate("serverConfigurationDialog", "OK"))
 
+
+if __name__ == "__main__":
```

Comment:
Were these automatically generated using the -x option to pyuic5 ? if so, thanks. I didn't know that an option like that existed! Can you also add them to pavement.py on lines 14 and 15 ? I generally run `paver mk_resources` to generate the artifacts. This would need a separate pip install step locally.

Reply:
I actually found out about this command [on StackOverflow](https://stackoverflow.com/a/43107732/3328268), but I didn't see it added this main. Not sure it's relevant here… the main program should be placed in `buildnotifyapplet.py` instead, shouldn't it?

Reply:
The actual main program is being triggered from the buildnotifyapplet.py. I  thought you had introduced these here using `-x` so that they can be opened directly. If you aren't using it then we can generate without `-x` (which would remove the __main__ block) .

I'm fine either ways. I think i'll merge it in the current state.

Reply:
Ok, thanks for this hint, I'll take in in account for the next PRs :)